packages/react/src/Token/Token.tsx

```diff
@@ -40,6 +44,7 @@ const Token = forwardRef((props, forwardedRef) => {
     onRemove,
     id,
     leadingVisual: LeadingVisual,
+    removeButtonLabel = 'Remove token',
```

Comment:
Just wondering, is this a safe default? Where are Tokens used in dotcom and would it be obvious what "token" is referring to?

Open questions: 
- Is no default better than confusing default?
- Is there a better default like `Remove ${props.text}`?

Reply:
Yup that's fair! I only kept it as `Remove token` since that was the default aria-label on it. We can definitely customize it a bit more, `Remove "${props.text}" token`.

> Is no default better than confusing default?

Yeah, making consumers add the label themselves is sometimes the better option. In this case, since it's already using an `aria-label="Remove token"`, I'm thinking the most backwards compatible way is to make it customizable while keeping the default? 🤔 

> Where are Tokens used in dotcom and would it be obvious what "token" is referring to?

Usage looks pretty limited [~ 3 instances](https://musical-adventure-wlr3n3k.pages.github.io/?name=token&attribute=onremove), which is good! I think by adding `${props.text}`, the "token" should be described in a way that users will know what it is.